### DISCUSSION TITLE
Syncing papis libraries on multiple computers

### DISCUSSION THREAD

**Discussion by Halem998** [papis/papis#587] [Q&A] [answered]
I'd like use my database on multiple computers.
If I choose a service like dropbox, what way to do so would be recommended?
Like, should I make use of `git` or should I just go with the default syncing capabilities of dropbox ... ?

Any suggestions appreciated :)

**Comment by jghauser** [comment, marked as answer]
I'm personally using syncthing for this, and it works very well. Dropbox should also do the trick. I'm not sure git would be the right tool for this use case, given all the large (non-plaintext) pdf/epub/etc files.

**Comment by alexfikl** [comment]
I basically use both for different reasons:
* each library gets a `git` repository to keep track of changes and plaintext files (`info.yaml`, notes, etc.). This ignores the large files like @jghauser said.
* each library is also backed up on `pcloud` to store the big PDF/DjVu/etc files. 

I haven't actually tried this setup on multiple computers though (just switching to a new machine), so not sure how well it would work, but theoretically Dropbox/Syncthing/pcloud should handle the syncing just fine.

**Comment by alejandrogallo** [comment]
My 5 cents. I exclusively use git only. THe git service did not yet complain about the size of the repository. I `git add` everything, I do not care if they are binary blobs or otherwise, pdf files never change so that's ok for me.

I have the library cloned in 4 computers, and I do pull/push from time to time. This worked well sofar for me. What I would consider is having also syncthing on top of it, and then I would not even have to do pull/push, but still I want everything on a git repo like @alexfikl, that is good for my OCD ;)

Side note, I also have a git hook in order to generate a bibtex file with all the documents before every commit.
Maybe quite overkill but it's good so far.

**Comment by Halem998** [comment]
ok thanks, wiser now :) I'll go with the lazy variant and do away with git
